feat(themes): allow custom animations on popups

luriusTM opened the pull request:
- Open tooltip on volume change in media center
- Scrollability on volume controls
- Percent description on volume notifier

https://github.com/user-attachments/assets/6b23e560-2ea0-4976-b32a-7f0e5edfb944

- Popup appear styleability
- Popup disappear styleability
Current simplest style applied

https://github.com/user-attachments/assets/498803bb-2346-49db-8069-c32f67638922


Animation possibilities (weg): 

https://github.com/user-attachments/assets/c30fe619-f5ce-4c5b-8488-8a8eebd0a199

Animation possibilities (tb): 

https://github.com/user-attachments/assets/f12f1d60-4acb-4c58-b5c7-fc55d4727d3e

eythaann commented:
Will be merged to v2.1, but disabling motion on the root config also disable animation on buttons, context menu and any other antd component? (it doesn't matter because all these will be replaced by self components latter but try it to know)

luriusTM commented:
> Will be merged to v2.1, but disabling motion on the root config also disable animation on buttons, context menu and any other antd component? (it doesn't matter because all these will be replaced by self components latter but try it to know)

It is hard to say what does it change for a button or slider... because it does not have too much animation... but this setting centrally removes all antd animation for the current scope (tb, weg). It removes from dropdown, that is for sure, it has same root appearance animation like popovers, also removes the minimal from tooltips. For calendar view it does not have any animation, 

Will replace the dropdown animation also then.

luriusTM commented:
- merged the master
- added context menus for the scope
- fixed context menu for launcher

pre: 
![image](https://github.com/user-attachments/assets/a5c5ff42-1e81-4617-a164-54dbf36720ef)
post: 
![image](https://github.com/user-attachments/assets/c7297aca-9afd-44bd-b278-8b56206e73f9)
